src/js/components/ErrorsAlert.js

```diff
@@ -4,8 +4,17 @@ import Alert from "./Alert";
 import { getUnanchoredErrorMessage } from "../utils/ErrorMessageUtil";
 
 const ErrorsAlert = function(props) {
-  const { errors, hideTopLevelErrors, pathMapping } = props;
-  let showErrors = errors;
+  const {
+    errors,
+    hideTopLevelErrors,
+    hidePermissiveErrors,
+    pathMapping
+  } = props;
+  let showErrors = [].concat.apply([], errors);
```

Comment:
Why is this needed. Default prop type is `[]` so this seems unneeded.

Reply:
@ahoskins are you saying the default `errors` prop is `[]`?

Reply:
I do see the case to keep `[].concat.apply([], errors)` in case `errors` is `undefined` or `null` which won't throw error when `showErrors.map..` or alike calls

Reply:
Okay, fair enough. When it's `undefined` no issue because it'll use the default prop. But to protect against `null` this is needed. 👍